frontend/src/views/camp/Dashboard.vue

```diff
@@ -300,6 +300,8 @@ export default {
     },
   },
   async mounted() {
+    this.api.reload(this.camp())
```

Comment:
If you don't await this, is this still helpful?

Reply:
In my testings it helped, but might be a timing issue. I didn't wanted to await the reload, otherwise you have to wait for the reload, whenever you navigate back to the dashboard.

Alternatively we would need to await for days and campCollaborations separately.

Reply:
Okay, let's merge then and see how it works, and possibly improve later